client/src/components/userAuctions/UserAuctions.jsx

```diff
@@ -36,7 +36,72 @@ class UserAuctions extends Component {
   }
 
   render() {
-    const { openAuctions, closedAuctions, isFetching } = this.props.userAuctions;
+    let { openAuctions, closedAuctions, isFetching } = this.props.userAuctions;
+    closedAuctions = [
```

Comment:
These are dummy data for closedAuctions, right? If so, it may be a good idea to put the db query results here.